Scarb.toml

```diff
@@ -1,15 +1,13 @@
-[package]
-name = "raito"
-version = "0.1.0"
-edition = "2024_07"
-
-[scripts]
-regenerate_tests= "./scripts/data/regenerate_tests.sh"
-integration_tests = "scarb build && ./scripts/data/integration_tests.sh"
-client= "scarb build && ./scripts/data/client.sh"
-test = "scarb cairo-test && scarb run integration_tests"
+[workspace]
+members = ["crates/*"]
```

Comment:
Scarb documentation uses term "package", Alexandria also uses packages instead of crates.